renovate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "reviewersFromCodeOwners": true,
+  "enabledManagers": [
+    "gomod",
+    "github-actions",
+    "dockerfile",
+    "custom.regex"
+  ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^GO_VERSION$"
+      ],
+      "depTypeTemplate": "language",
+      "matchStrings": [
+        "^(?<currentValue>.+)\\n"
```

Comment:
This newline should be fixed similar to the one in gotify/build. Otherwise it looks good (renovate output)
```
                 "depName": "gotify/build",
                 "currentValue": "1.22.4",
                 "datasource": "github-tags",
                 "extractVersion": "^v(?<version>.+)$",
                 "depType": "language",
                 "replaceString": "1.22.4\n",
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "1.23.2",
                     "newValue": "1.23.2",
                     "releaseTimestamp": "2024-11-01T21:04:27.000Z",
                     "newVersionAgeInDays": 0,
                     "newMajor": 1,
                     "newMinor": 23,
                     "newPatch": 2,
                     "updateType": "minor",
                     "branchName": "renovate/gotify-build-1.x"
                   }
                 ],

```